applications/views.py

```diff
@@ -29,7 +29,7 @@ def get_queryset(self):
         if self.action == "retrieve":
             return BootcampApplication.objects.prefetch_state_data()
         else:
-            return BootcampApplication.objects.all()
+            return BootcampApplication.objects.filter(user=self.request.user).all()
```

Comment:
Should we do this filtering for `retrieve` as well?

Reply:
That's handled through the `main.permissions.UserIsOwnerPermission` permission class (the view returns a 403 if you're requesting an application that isn't yours)